luigi/contrib/hadoop.py

```diff
@@ -400,7 +400,7 @@ class HadoopJobRunner(JobRunner):
 
     def __init__(self, streaming_jar, modules=None, streaming_args=None,
                  libjars=None, libjars_in_hdfs=None, jobconfs=None,
-                 input_format=None, output_format=None,
+                 archives=None, input_format=None, output_format=None,
```

Comment:
Well, as it is a  a generic option I thought it would be better next to another one like, like libjars. But I guess I should have added it on the top. Then, I will update the PR to put it at the end.